frontend/src/components/BoosterButton.js

```diff
@@ -1,36 +1,57 @@
-import React from 'react';
+import React, {useEffect, useState} from 'react';
 import Button from "@material-ui/core/Button";
 import makeStyles from "@material-ui/core/styles/makeStyles";
 import Grid from "@material-ui/core/Grid";
-import Link from "@material-ui/core/Link";
+import {fetchIdFromType} from "../utils/booster-utils";
+import {useHistory} from "react-router";
 
+const owner = "2";
 const useStyles = makeStyles((theme) => ({
     button: {
-        width: 325,
-        height: 160,
+        width: 120,
+        height: 60,
         margin: "20px 0px",
         borderRadius: "10%"
     },
 
-
 }));
 
-export default function BoosterButton(props) {
+
+export default function BoosterButton() {
+    const [randomId, setRandomId] = useState([]);
     const classes = useStyles();
-    return (
+    const history = useHistory();
+
 
 
+    function redirect(boosterType,owner) {
```

Comment:
I think it's a problem that this function gets automatically called which results somehow in an infinite fetch loop